index.html

```diff
@@ -0,0 +1,424 @@
+        <div class="headline-container">
+
+          <div class="heads-container">
+            <div class="heading-1">
+              <h1>Hey There. I'm Zunair</h1>
+            </div>
+            <div class="heading-2">
+              <h2>I'am a software developer</h2>
+            </div>
+            <div class="paragraph">
+              <p>
+                I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and have
+                a project you need coded,don’t hestiate to contact me.
+              </p>
+            </div>
+          </div> 
+
+          <div class="social-icons-container">
+            <ul>
+              <li>
+                <a href="#"
+                  ><i class="fa fa-github social-icons" aria-hidden="true"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><i class="fa fa-linkedin social-icons" aria-hidden="true"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><i
+                    class="fa fa-angellist social-icons"
+                    aria-hidden="true"
+                  ></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><i class="fa fa-twitter social-icons" aria-hidden="true"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#" class="vectorimg"
+                  ><img
+                    src="./images/Vector.png"
+                    class="vectorimg"
+                    alt="Medium-icon"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="illustration-header">
+          <img src="/images/Illustration-header.png" alt="">
+        </div> 
+      </section>
+
+
+      <section class="work-section" id="portfolio">
+        <div class="my-recentwork-container">
+            <div class="box-1">
+              <h1>My Recent Works</h1>
+            </div>
+            <div class="box-1-line"></div>
+        </div>
+
+        
+            <div class="multipost-boxes-container">
+              <div class="boxes-1">
+               
+                 <a href="#"
+                   ><img src="./images/imageplaceholder4.png" alt="Works-image"
+                 /></a>
+              </div>
+     
+              <div class="boxes-2">
+
+                 <div class="multi-post-heading">
+                   <h3>Multi-Post Stories</h3>
+                 </div>
+                 <div class="multi-post-para">
+                   <p>
+                     A daily selection of privately personalized reads; no accounts or
+                     sign-ups required. has been the industry's standard dummy text
+                     ever since the 1500s, when an unknown printer took a standard
+                     dummy text.
```

Comment:
- [ ] I have noticed that the `box` is not yet stable. On hover, the box messes up 😞 . This can frustrate the user when interacting with your site. Kindly fix that by referring to the previous reviewer's suggestion about this 👍 .

Reply:
The boxes are stable now